.golangci.yml

```diff
@@ -1,15 +1,3 @@
-######################################## {COPYRIGHT-TOP} ###
-# Licensed Materials - Property of IBM
-#
-# 5900-A0Y, 5737-H57
-#
-# (C) Copyright IBM Corp. 2020 All Rights Reserved.
-#
-# US Government Users Restricted Rights - Use, duplication or
-# disclosure restricted by GSA ADP Schedule Contract with
-# IBM Corp.
-######################################## {COPYRIGHT-END} ###
-# DO NOT EDIT: Generated by https://github.ibm.com/mqoc/repo-bootstrap
 run:
   # timeout for analysis, e.g. 30s, 5m, default is 1m
```

Comment:
is this file still needed?

i dont find any reference on sdk template repo or sample sdk repos.

worth a check.